Email/email_reputation_checker.py

```diff
@@ -54,7 +54,7 @@ def checkEmailBreach(self, email):
         except Exception as e:
             utils.error_message(e.text)
         if response.status_code == 404:
-            logger.debug("{}".format(colored("No breaches found for this email","green")))
+            logging.debug("{}".format(colored("No breaches found for this email","green")))
```

Comment:
Fixes https://github.com/initd1/HackAlert/pull/24#issuecomment-1501174640

Reply:
Nice work. Is this PR ready for review?

Reply:
yes. But I couldn't fix the issue of not logging to traceback.log.  Please create a separate issue if you are able to look into it. thanks